pkg/protodep/gomod.go

```diff
@@ -0,0 +1,331 @@
+package protodep
+
```

Comment:
Would have expected this library in a standalone PR, this PR is already quite large yet the testing for the library seems too minimal if we’re to use this everywhere